core/pom.xml

```diff
@@ -68,6 +68,10 @@
             <groupId>org.eclipse.microprofile.openapi</groupId>
             <artifactId>microprofile-openapi-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+        </dependency>
```

Comment:
Core should not have a dependency on `hibernate-validator`.

What's the reason for `CreateClient` and `CreateClientDefault` to be in core? Is it not just used server-side by the Client v2 implementation?